src/quest/historian.py

```diff
@@ -192,7 +193,7 @@ def _get_exception_class(exception_type: str):
 
 
 class Historian:
-    def __init__(self, workflow_id: str, workflow: Callable, history: History):
+    def __init__(self, workflow_id: str, workflow: Callable, history: History, serializer: StepSerializer = None):
```

Comment:
Let's take off the default argument for serializer and make serializer required.

You'll need to update all the tests that use Historian to use the NoopSerializer